.husky/pre-commit

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh
```

Comment:
Is the shebang removed unexpectedly?

Reply:
Agh, yup! Just resolved it. Late night for me, so maybe I'll stop making any big changes!